pkg/code-generator/codegen/conversion.go

```diff
@@ -0,0 +1,165 @@
+package codegen
+
+import (
+	"bytes"
+	"sort"
+	"text/template"
+
+	"github.com/solo-io/go-utils/versionutils/kubeapi"
+	"github.com/solo-io/solo-kit/pkg/errors"
+
+	"github.com/solo-io/go-utils/log"
+	code_generator "github.com/solo-io/solo-kit/pkg/code-generator"
+	"github.com/solo-io/solo-kit/pkg/code-generator/codegen/templates"
+	"github.com/solo-io/solo-kit/pkg/code-generator/model"
+)
+
+func GenerateConversionFiles(soloKitProject *model.ApiGroup, projects []*model.Version) (code_generator.Files, error) {
+	var files code_generator.Files
+
+	sort.SliceStable(projects, func(i, j int) bool {
+		vi, err := kubeapi.ParseVersion(projects[i].VersionConfig.Version)
```

Comment:
Could be cool to make this sort function a function in `kubeapi` which takes in the 2 versions and returns the function. So a lambda I guess

Reply:
agreed, but let's hold off till this comes up again.